static/app/views/metrics/billing.tsx

```diff
@@ -0,0 +1,46 @@
+import Alert from 'sentry/components/alert';
+import {LinkButton} from 'sentry/components/button';
+import HookOrDefault from 'sentry/components/hookOrDefault';
+import useOrganization from 'sentry/utils/useOrganization';
+
+export const MetricsSubscriptionCTAProvider = HookOrDefault({
+  hookName: 'component:metrics-subscription-cta',
+  defaultComponent: () => null,
+});
+
+export function TopBanner() {
+  const organization = useOrganization();
+
+  return (
+    <MetricsSubscriptionCTAProvider organization={organization}>
+      {subscriptionCTAProps => {
+        if (!subscriptionCTAProps) {
+          return null;
+        }
+        return (
+          <Alert system type={subscriptionCTAProps.type}>
+            {subscriptionCTAProps.description}
+          </Alert>
+        );
+      }}
+    </MetricsSubscriptionCTAProvider>
+  );
```

Comment:
nit, we usually just call the hook and let getsentry do all the business logic of rendering or not rendering components